### PR TITLE
Hamdhan Junit tests and add assert statements

### PR DESCRIPTION
**Pull request by mohammedhamdhan**

### CODE REVIEW

**Review by matthewyeo1 — COMMENTED** [review]
Good coverage of JUnit tests!

File: src/main/java/seedu/duke/commands/ExpenseCommand.java [hunk 1]
```diff
@@ -111,17 +126,30 @@ public void executeDeleteExpense() {
...
+            String indexStr = scanner.nextLine().trim();
+            
+            if (indexStr.isEmpty()) {
+                System.out.println("Please enter a valid expense number.");
+                return;
+            }
+            
+            int index = Integer.parseInt(indexStr) - 1; // Convert to 0-based index
+            
+            if (index < 0 || index >= budgetManager.getExpenseCount()) {
```
Comment (NandhithaShree) [inline src/main/java/seedu/duke/commands/ExpenseCommand.java:138]: Great Job handling edge cases!

File: src/test/java/seedu/duke/commands/ExpenseCommands.java [hunk 2]
```diff
@@ -208,5 +208,51 @@ void testDisplayUnsettledExpensesTwoUnsettledExpenses(){
         assertEquals(expectedMessage, actualOutput);
     }
     //@@author
+
+    //@@author mohammedhamdhan
+    @Test
+    void testExecuteDeleteExpenseInvalidInput() {
```
Comment (NandhithaShree) [inline src/test/java/seedu/duke/commands/ExpenseCommands.java:214]: great job naming the tests like shown in the lecture

**Review by Ashertan256 — COMMENTED** [review]
Great job, can't wait for these changes to be merged!
